omelet-core/src/main/java/omelet/data/xml/MappingParserRevisit.java

```diff
@@ -158,6 +158,13 @@ public static String getBuildNumber() {
 		return getCalcValue("buildNumber");
 	}
 
+	public static String getMappinfFile() {
```

Comment:
small typo in the message name ,
However just one thing have you tested this with no key provided at all ?
Did it work ?
As there are no test cases for this but atleast manual check can do at this point